operations/helm/charts/mimir-distributed/values.yaml

```
@@ -25,21 +25,24 @@ image:
   #   - myRegistryKeySecretName
 
 global:
-  # definitions to set up nginx resolver
+  # -- Definitions to set up nginx resolver
   dnsService: kube-dns
   dnsNamespace: kube-system
   clusterDomain: cluster.local
 
-  # Common environment variables to add to all PODs directly managed by this chart (mimir services, nginx). Doesn't include memcached or minio.
+  # -- Common environment variables to add to all PODs directly managed by this chart (mimir services, nginx).
   extraEnv: []
 
-  # Common source of environment injections to add to all PODs directly managed by this chart (mimir services, nginx). Doesn't include memcached or minio.
+  # -- Common source of environment injections to add to all PODs directly managed by this chart (mimir services, nginx).
   # For example to inject values from a Secret, use:
   # extraEnvFrom:
   #   - secretRef:
   #      name: mysecret
   extraEnvFrom: []
 
+  # -- POD annotations for all PODs directly managed by this chart (mimir services, nginx). Useable for example to associate a version to 'global.extraEnv' and 'global.extraEnvFrom' and trigger a restart of the affected services.
```

```suggestion
  # -- Additional pod annotations for all Pods directly managed by this chart (mimir services, nginx, memcached; not minio). Usable for example to associate a version to 'global.extraEnv' and 'global.extraEnvFrom' and trigger a restart of the affected services.
```

I would clarify which pods this includes. A user might not look at the chart implementation to figure out we use a subchart to deploy minio. Although I see how this can get out of date pretty quickly. Not sure what's a good strategy to prevent this. Maybe a place where we declare all resources we deploy and which chart manages them? IDK


What's the reason for using `POD` instead of `pod`?

---

POD because I forgot that it's a word, not an abbreviation.

Yeah, the list is tricky. Memcached is not actual part of that list at the moment, only after #2064 (I need to comment on it if this is merged before). I'll add the full list, on it's own line, so we could potentially write some test for it (cc: @Logiraptor )

---

Added commit to be more explicit and fix spelling.